packages/@react-spectrum/table/src/Table.tsx

```diff
@@ -384,10 +410,11 @@ function TableColumnHeader({column}) {
             )
           )
         }>
-        {column.rendered}
+        {!columnProps.hideHeader && column.rendered}
```

Comment:
Since TooltipTrigger should not add `aria-describedby` until the Tooltip is open, we should add the rendered column header content within a `VisuallyHidden` element:
```jsx
        {columnProps.hideHeader ?
          <VisuallyHidden>{column.rendered}</VisuallyHidden> :
          column.rendered
        }
```

Reply:
Related to this, `TooltipTrigger` is always adding `aria-describedby` regardless of whether the `state.isOpen` or not.

At:
https://github.com/adobe/react-spectrum/blob/975957a3c3fb4a9f3be358428e0980a4d4e51c1a/packages/%40react-aria/tooltip/src/useTooltipTrigger.ts#L135
`state.open` is a `function`, so it's always `true`, whereas `state.isOpen` is a `boolean`, this line should be:
```jsx
      'aria-describedby': state.isOpen ? tooltipId : undefined,
```

Reply:
I've opened a related issue: https://github.com/adobe/react-spectrum/issues/1149.

Reply:
Made the required changes
cc: @snowystinger @majornista